documents/knowledge/elements.yaml

```diff
@@ -1444,12 +1444,14 @@
   arity: 1
   overloads:
     any: bool(a) (Booliify)
-  vectorise: true
+  vectorise: false
```

Comment:
add a note here that its behaviour is changed by the `t` flag?

Reply:
as a comment?

Reply:
like that?